src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -3505,10 +3505,12 @@ public DatabaseTopology ReadDatabaseTopology<TTransaction>(TransactionOperationC
         {
             using (var databaseRecord = ReadRawDatabaseRecord(context, name))
             {
+                DatabaseTopology topology;
                 if (databaseRecord.IsSharded)
-                    throw new InvalidOperationException($"The database record '{name}' is sharded and doesn't contain topology directly.");
+                    topology = databaseRecord.Sharding?.Orchestrator.Topology;
```

Comment:
It is make sense, but I think we rely on this to catch wrong usage of getting topology for a shard.
why do you need this change?

Reply:
Revert changes